src/utils.js

```diff
@@ -8,3 +13,21 @@ export const formatTime = (date) => {
 
   return `${hours}:${minutes}`;
 };
+
+export const createElement = (template) => {
+  const newElement = document.createElement(`div`);
+  newElement.innerHTML = template;
+
+  return newElement.firstChild;
+};
+
+export const render = (container, element, place) => {
```

Comment:
Все сделал отлично!
Надо поправить - кавычки во всех файлах вразнобой.
И можно улучшить - 
```suggestion
export const render = (container, element, place = RenderPosition.BEFOREEND) => {
```
Тогда не нужно будет ее передавать каждый раз, а только когда вставляешь в начало.